ingest-app/src/cmr/ingest/api/ingest/bulk.clj

```diff
@@ -0,0 +1,122 @@
+(ns cmr.ingest.api.ingest.bulk
```

Comment:
Yeah, I've been rebasing from master regularly -- will do again before merge. I'll ping @laurenfrederick here so she can give it a once-over and make sure nothing she's added has been lost in the merges/rebases.